source/code/source/playfab/PlayFabWinHttpPlugin.cpp

```diff
@@ -255,7 +255,7 @@ namespace PlayFab
                                         }
 
                                         // Allocate space for the buffer
-                                        auto outBuffer = std::unique_ptr<char[]>(new char[dwSize + 1]);
+                                        std::unique_ptr<char[]> outBuffer = std::unique_ptr<char[]>(new char[dwSize + 1]);
```

Comment:
>std::unique_ptr<char[]> [](start = 40, length = 23)

the type is clearly stated on the right side. I don't think removing the auto here really helps much. again I'll leave it to your style discretion but I think leaving this as auto would be fine